### PR TITLE
Aws feature branch to merge

### PR DESCRIPTION
**Pull request by LadunOmideyiSoftwire**
Notes:
- Terraform files have to be at the root so can't be organised
- Locals work like variables and variables like environment variables
- We removed main 
- Logging and monitoring are located with their respective resources except for alarms
- Any value that has be changed has a comment next to it

### CODE REVIEW

File: terraform/elasticbeanstalk.tf [hunk 1]
```diff
@@ -1,98 +1,336 @@
...
+    name      = "AccessLogsS3Prefix"
+    value     = local.env_prefix
+  }
+
+  // HTTP listener config
+
+  setting {
+    namespace = "aws:elbv2:listener:default"
+    name      = "ListenerEnabled"
+    value     = "false"  // disabled. we create out own port 80 listener which redirects to https
```
Comment (davidm-m) [inline terraform/elasticbeanstalk.tf:143]: Typo: "we create ou**r** own port 80 listener"

File: terraform/loadtest.tfvars [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+elb_deployment_policy = "Rolling"
+elb_instance_max_size = 2
+elb_instance_min_size = 1
+elb_instance_type     = "t2.small"
+
+#endregion
+
+#region Cloudfront configuration
+
+cloudfront_alternate_domain_name = "ladun.me"
```
Comment (davidm-m) [inline terraform/loadtest.tfvars:24]: I think this one needs a reminder to change it

File: terraform/loadtest.tfvars [hunk 3]
```diff
@@ -0,0 +1,26 @@
+env     = "loadtest"
```
Comment (davidm-m) [inline terraform/loadtest.tfvars:1]: Is the plan to have a load test environment running all the time?

File: terraform/locals.tf [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (davidm-m) [inline terraform/locals.tf:1]: Is this file doing anything? 😅

File: terraform/variables.tf [hunk 5]
```diff
@@ -233,17 +183,27 @@ variable "ELB_WEBJOBS_STOPPED" {
...
+
+#endregion
+
+#region cloudwatch config
+
+variable "CLOUDWATCH_NOTIFICATION_EMAILS" {
   type        = string
-  description = "AWS secret access key. Set as an environment variable."
+  description = "An email distribution list to be notified of alarm breaches. Pass in as environment variable."
+  default     = "ladun.omideyi@softwire.com"
```
Comment (davidm-m) [inline terraform/variables.tf:206]: Can we make this the team email instead?

**Review by davidm-m — CHANGES_REQUESTED** [review]
A few things I want to check up on. One thing I haven't commented on but would be nice is if all of the code comments like "change this on production" had some identifiable string (`QQ` is sometimes used for things like this) that's easy to search the codebase for so we know we've made all the necessary changes before putting this live.

File: GenderPayGap.WebUI/wwwroot/robots.txt [hunk 6]
```diff
@@ -1,25 +1,2 @@
...
-Disallow: /viewing/azure-uptime-pinger
-Disallow: /viewing/add-search-results-to-compare
-
-User-agent: SemrushBot
-Disallow: /
-
-User-agent: SemrushBot/2
-Disallow: /
\ No newline at end of file
+Disallow: /*
```
[the remark this answers is not in the document]
Comment (LadunOmideyiSoftwire) [reply GenderPayGap.WebUI/wwwroot/robots.txt:2]: For the migration to make files non indexable. Removing as the site is going live tomorrow

**Review by davidm-m — APPROVED** [review]

**Pull request merged**